valhalla/baldr/graphconstants.h

```diff
@@ -132,7 +132,8 @@ enum class RoadClass : uint8_t {
   kTertiary = 4,
   kUnclassified = 5,
   kResidential = 6,
-  kServiceOther = 7
+  kServiceOther = 7,
+  kInvalidRoadClass = 8, // only 3 bits in DE
```

Comment:
not useful for directed edges, but there's quite a few places where we'd like to have smth like this